server/models/AssessmentVariablesModel/assessmentVariablesModel.test.js

```diff
@@ -0,0 +1,186 @@
+import AssessmentVariablesModel from '.'
+import { createAsessmentVariable } from '../../../test/fixtures'
+import { collections } from '../../utils/mongoCollections'
+
+describe('AssessmentVariablesModel', () => {
+  let appDb
+
+  const initialAssessmentVariables = [
+    createAsessmentVariable({
+      name: 'var1',
+    }),
+    createAsessmentVariable({
+      name: 'var2',
+    }),
+    createAsessmentVariable({
+      name: 'var3',
+    }),
+    createAsessmentVariable({
+      name: 'var4',
+    }),
+    createAsessmentVariable({
+      name: 'var6',
+    }),
+    createAsessmentVariable({
+      name: 'var7',
+    }),
+  ]
+  beforeAll(async () => {
+    appDb = await global.MONGO_INSTANCE.db('assessment_vars')
+  })
+  beforeEach(async () => {
+    await appDb.createCollection(collections.assessmentVariables)
+  })
+
+  afterEach(async () => {
+    await appDb.collection(collections.assessmentVariables).drop()
+  })
+  afterAll(async () => {
+    await appDb.dropDatabase()
+  })
+
+  it('creates new assessment variables', async () => {
```

Comment:
this description says that it creates new variables but the assertion expects the length of the variables to be unchanged. Is this description wrong?

Reply:
I made the changes to the test